### PR TITLE
teams/docs: a vote for Oreoxmt as a reviewer

### PR DESCRIPTION
**Pull request by ran-huang**
As stated in the title.

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM
Congratulations!

**Review by qiancai — APPROVED** [review]
LGTM

Thanks for the excellent work. Congratuations.

File: votes/0660-Oreoxmt-as-docs-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Deadline
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team: Docs
+
+## Result
```
Comment (ran-huang) [inline votes/0660-Oreoxmt-as-docs-reviewer.md:28]:
```suggestion
## Result

Approved by 2 binding votes:

* qiancai (binding)
* TomShawn (binding)
```

**Comment by ran-huang** [comment]
@zhangyangyu PTAL, thanks. Can you help merge this PR?

**Pull request merged**